.github/actions/checkov-scans-v1/action.yml

```
@@ -0,0 +1,40 @@
+name: checkov-scans
+description: Checkov scans cloud infrastructure configurations to find misconfigurations before they're deployed and sends a result to Prisma Cloud.
+
+inputs:
+  output-file-path:
+    description: Output file path of Checkov scans
+    required: true
+  prisma-access-key:
+    description: Prisma Cloud access key
+    required: true
+  prisma-secret-key:
+    description: Prisma Cloud secret key
+    required: true
+  prisma-api-url:
+    description: Prisma Cloud API URL
+    required: true
+
+runs:
+  using: 'composite'
+  steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so follow-up steps can access it
+    - uses: actions/checkout@v4
+    # Used to name the file from Checkov
+    - name: Extract package version
+      id: extract-version
+      shell: bash
+      run: echo "::set-output name=version::$(jq -r .version package.json)"
```

This way of setting outputs for GHA's has been deprecated for a while (see [article](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/)). There are various examples in the actions directory of apply the new format
```suggestion
			run: echo version=$(jq -r .version package.json) >> "$GITHUB_OUTPUT"
```
The root package JSON doesn't normally update it's version when releasing. We have a script that does get reused quite regularly and there is a tech-debt ticket to turn that into it's own GHA. For now I think it's fine to reuse the script and reference the ticket so we can eventually pay that debt back: https://github.com/dequelabs/axe-api-team-public/issues/115. 

I don't actually see for example: `steps.extract-version.outputs.version` be used in the actual prisma cloud action, do we even need it or can we just run prisma?

---

This is not used because we don't need a PR after that. I'll delete it

---

@Zidious 
deleted this step